### DISCUSSION TITLE
F-Droid Caveats

### DISCUSSION THREAD

**Discussion by redstreet** [MM2-0/Kvaesitso#281] [Q&A]
Hello there,
F-Droid includes the following caveats, and I'm wondering if at all if would be possible to include a line or two in the app description (or README) to address why these caveats exist:

- [The upstream source code is not entirely Free](https://f-droid.org/en/docs/Anti-Features/#UpstreamNonFree)

- [This app promotes or depends entirely on a non-free network service](https://f-droid.org/en/docs/Anti-Features/#NonFreeNet)

This would allow users to understand how these warnings apply to Kavesitso. Thanks much!

**Comment by MM2-0** [comment]
> The upstream source code is not entirely Free

I have no idea why they think that.

> This app promotes or depends entirely on a non-free network service

This is because of the integrations of the OpenWeatherMap / Met.no / BrighSky / HERE weather APIs, the ECB currency API and the Google Drive and OneDrive file APIs.
